src/screens/HomeScreen/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Navbar from '../../components/Navbar';
+import Header from '../../components/Header';
+import List from './components/List';
+import { getCountries } from '../../Redux/homeSlice';
+
+function HomeScreen() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getCountries());
+  }, []);
+
+  const { countries } = useSelector((state) => state.countries);
+  let totalPopulation = 0;
+  for (let i = 0; i < countries.length; i += 1) {
+    totalPopulation += JSON.parse(countries[i].population);
+  }
+  return (
+    <div className="homeScreen">
+      <Navbar />
+      <Header population={totalPopulation} />
+      <List data={countries} />
+    </div>
+  );
+}
```

Comment:
- [ ] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name.

Reply:
Hello @Deepakdanger, 
**Hope everything is going great by your side**

These are the project requirements that we were given for the home page: 

- When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.

![HomePage](https://user-images.githubusercontent.com/66283922/194389126-f906561b-241f-4374-b4a6-fecdbac0bece.PNG)

**As you can see in the screenshot from the project we can click on a category which is a card then we are taken to the details screen where the app will show details about that page.**

Kindly check this requirement once more.